packages/bitcoin/src/coin-selection/calculate-max-spend.ts

```diff
@@ -1,22 +1,24 @@
-import type { AverageBitcoinFeeRates, BitcoinAddress, Money } from '@leather.io/models';
-import { createMoney } from '@leather.io/utils';
+import BigNumber from 'bignumber.js';
 
-import { CoinSelectionUtxo } from '../coin-selection/coin-selection';
-import {
-  filterUneconomicalUtxos,
-  getSpendableAmount,
-} from '../coin-selection/coin-selection.utils';
+import type { AverageBitcoinFeeRates, Money } from '@leather.io/models';
+import { createMoney, satToBtc } from '@leather.io/utils';
+
+import { CoinSelectionUtxo } from './coin-selection';
+import { filterUneconomicalUtxos, getSpendableAmount } from './coin-selection.utils';
 
 interface CalculateMaxSpendArgs {
-  recipient: BitcoinAddress;
+  // recipient is intentionally string instead of BitcoinAddress, as it's being validated
```

Comment:
I think this validation is done in the `mobile/src` right? So it doesn't make sense to add this comment to the bitcoin package?

If it needs to be a string that's OK but we should keep this package as an independent entity

Reply:
Perhaps the comment is a bit vague, "validated in subroutine" means `getSpendableAmount` validates the address, so it doesn't have to yet be valid at this stage.